services/state/file/file.proto

```diff
@@ -29,39 +38,77 @@ option java_package = "com.hederahashgraph.api.proto.java";
 option java_multiple_files = true;
 
 /**
- * Representation of a Hedera Token Service file in the network Merkle tree.
+ * Representation of an Hedera File Service `file`.
+ *
+ * Files offer a place to store additional data, much more than is available in
+ * other entities, for use with smart contracts, non-fungible tokens, etc...
+ * As with all network entities, a file has a unique entity number, which is
+ * given along with the network's shard and realm in the form of a
+ * shard.realm.number id.
  *
- * As with all network entities, a file has a unique entity number, which is given along
- * with the network's shard and realm in the form of a shard.realm.number id.
+ * > Review Question
+ * >> File lacks auto renewal (account and period).  We need to add that here,
+ * >> and also to create and update transactions.
+ * >
+ * >> The keys field is weird and really needs to change to adminKey and
```

Comment:
@povolev15 Do you agree that keys should change to adminKey and deleteKey? Even if it should change, this is outside the scope of this documentation update. If you think it should change then please create an issue.